llvm/docs/ReleaseNotes.md

```diff
@@ -447,6 +447,9 @@ Changes to the LLVM tools
 Changes to LLDB
 ---------------------------------
 
+* When building LLDB with Python support, the minimum version of Python is now
+  3.8.
+
```

Comment:
This looks weird now but when I rebase after the branching, this will be the one and only note here.